paymentsheet-example/build.gradle

```diff
@@ -52,4 +55,20 @@ android {
     buildFeatures {
         viewBinding true
     }
+}
+
+check.dependsOn('detekt')
+
+detekt {
+    buildUponDefaultConfig = true
+    allRules = true
+    config = files("$projectDir/../config/detekt.yml")
+
+    reports {
+        html.enabled = true // observe findings in your browser with structure and code snippets
+    }
+}
+
+tasks.withType(io.gitlab.arturbosch.detekt.Detekt) {
+    autoCorrect = true
 }
```

Comment:
Can you move this to the top-level `build.gradle` so we don't need to repeat it everywhere?